shared/core/types/pools.go

```diff
@@ -1,18 +1,35 @@
 package types
```

Comment:
Was doing something related in v0 this week: https://github.com/pokt-network/pocket-core/pull/1540/files

Reply:
LOL